### ISSUE TITLE
[Question] - Berksfile has dependency not stated in metadata.rb... why?

### ISSUE THREAD

**Issue by kquinsland** [clearstorydata-cookbooks/apache_spark#19]
Hey there guys!

I'm trying to diagnose an issue i see with our own internal wrapper cookbook for this cookbook.  We're seeing a very curious loop with the `monit` service

```
* script[monit-reload] action run
             - execute "bash"  "/tmp/chef-script20160512-1369-g6b3zv"
           * ruby_block[ensure-monit-is-running-after-reloading-for-spark-standalone-master] action run
             - execute the ruby block ensure-monit-is-running-after-reloading-for-spark-standalone-master
           * ruby_block[wait-for-monit-reload-spark-standalone-master] action run
             - execute the ruby block wait-for-monit-reload-spark-standalone-master
         Recipe: sysctl::default
           * ruby_block[save-sysctl-params] action run
             - execute the ruby block save-sysctl-params
         Recipe: monit-ng::service
           * service[monit] action restart
             - restart service service[monit]
           * service[monit] action start (up to date)
         Recipe: monit-ng::reload
           * ruby_block[conditional-monit-reload] action run
             - execute the ruby block conditional-monit-reload
         Recipe: apache_spark::spark-standalone-master
           * monit_wrapper_service[spark-standalone-master] action start
           Recipe: sysctl::default
             * ruby_block[save-sysctl-params] action run
        - execute the ruby block save-sysctl-params
           Recipe: monit-ng::service
             * service[monit] action restart
        - restart service service[monit]
             * service[monit] action start (up to date)
           Recipe: monit-ng::reload
             * ruby_block[conditional-monit-reload] action run
        - execute the ruby block conditional-monit-reload
```

and this repeats over and over again.  This leads me to **my question**:

This is the dependency graph that i was able to build:

  `apache_spark` (1.2.12) [DEPENDS->     `monit_wrapper` (~> 3.0)  [DEPENDS-> `monit-ng`

https://github.com/clearstorydata-cookbooks/apache_spark/blob/master/metadata.rb#L31
https://github.com/clearstorydata-cookbooks/monit_wrapper/blob/master/metadata.rb#L29
https://github.com/bbg-cookbooks/monit-ng/blob/master/metadata.rb

But when looking at the`Berksfile`, there's a specific repo and version defined for the `monit` and **not** the `monit-ng` cookbook

https://github.com/tfly-cookbooks/apache_spark/blob/master/Berksfile#L5

So basically, can somebody clearly explain why the `Berksfile` and `metadata.rb` file list different dependencies?
